zerver/lib/utils.py

```diff
@@ -17,8 +17,16 @@
 from six.moves import range
 from zerver.lib.str_utils import force_text
 
+if False:
+    # This needed for typing only
+    from zerver.models import Realm
+
 T = TypeVar('T')
 
+def name_changes_disabled(realm):
+    # type: (Realm) -> bool
+    return settings.NAME_CHANGES_DISABLED or realm.name_changes_disabled
```

Comment:
Let's move this to zerver/models.py instead; I don't think it really belongs in this file.